dashboard/src/components/Badge.tsx

```diff
@@ -37,12 +37,12 @@ export interface BadgeProps {
 }
 export default function Badge(props: BadgeProps): JSX.Element {
   const colorVariants = {
-    [BadgeCodes.ERROR]: "bg-failed text-white",
-    [BadgeCodes.WARNING]: "bg-[#ffa800] text-white",
-    [BadgeCodes.SUCCESS]: "bg-[#00c2ab] text-black-50 text-black-800",
+    [BadgeCodes.ERROR]: "bg-red-500 text-white",
+    [BadgeCodes.WARNING]: "bg-yellow-400 text-white",
+    [BadgeCodes.SUCCESS]: "bg-green-300 text-black-50 text-black-800",
     [BadgeCodes.INFO]: "bg-blue-200 text-black-800",
     [BadgeCodes.DEFAULT]: "text-black-800",
-    [BadgeCodes.UNKNOWN]: "text-failed bg-zinc-200",
+    [BadgeCodes.UNKNOWN]: "text-red-500 bg-zinc-200",
```

Comment:
no
1. these are not the right colors, why did you changed them?
2. use tailwind theme colors

Reply:
cuz @undera said they are the wrong colors in slack, mind telling me which colors to use plz.